RecoPixelVertexing/PixelTriplets/plugins/CAHitNtupletHeterogeneousEDProducer.cc

```diff
@@ -52,35 +52,33 @@ class CAHitNtupletHeterogeneousEDProducer
                   const edm::EventSetup &iSetup) override;
 
 private:
-  edm::EDGetTokenT<IntermediateHitDoublets> doubletToken_;
+  edm::EDGetTokenT<edm::OwnVector<TrackingRegion> > regionToken_;
 
   edm::EDGetTokenT<HeterogeneousProduct> tGpuHits;
 
-
   edm::RunningAverage localRA_;
   CAHitQuadrupletGeneratorGPU GPUGenerator_;
-  CAHitQuadrupletGenerator CPUGenerator_;
 
-  bool emptyRegionDoublets = false;
+  bool emptyRegions = false;
   std::unique_ptr<RegionsSeedingHitSets> seedingHitSets_;
 };
 
 CAHitNtupletHeterogeneousEDProducer::CAHitNtupletHeterogeneousEDProducer(
     const edm::ParameterSet &iConfig)
     : HeterogeneousEDProducer(iConfig),
-      doubletToken_(consumes<IntermediateHitDoublets>(
-          iConfig.getParameter<edm::InputTag>("doublets"))),
+      regionToken_(consumes<edm::OwnVector<TrackingRegion>>(
+          iConfig.getParameter<edm::InputTag>("trackingRegions"))),
       tGpuHits(consumesHeterogeneous(iConfig.getParameter<edm::InputTag>("heterogeneousPixelRecHitSrc"))),
-      GPUGenerator_(iConfig, consumesCollector()),
-      CPUGenerator_(iConfig, consumesCollector()) {
+      GPUGenerator_(iConfig, consumesCollector()) {
   produces<RegionsSeedingHitSets>();
 }
 
 void CAHitNtupletHeterogeneousEDProducer::fillDescriptions(
     edm::ConfigurationDescriptions &descriptions) {
   edm::ParameterSetDescription desc;
 
-  desc.add<edm::InputTag>("doublets", edm::InputTag("hitPairEDProducer"));
+  desc.add<edm::InputTag>("doublets", edm::InputTag("dummy"))->setComment("Not really used, kept to keep the python parameters");
```

Comment:
Can we set it to be empty by default, instead of using a dummy value ?
One of the checks we do at HLT is to avoid InputTag referencing non-existing modules, and these "dummy" labels end up giving false  positives.